public/js/SpriteSheet.js

```diff
@@ -38,24 +38,24 @@ export default class SpriteSheet {
             return buffer;
         });
 
-        this.tiles.set(name, buffers);
+        this.tiles.set(style, buffers);
     }
 
-    defineTile(name, x, y) {
-        this.define(name, x * this.width, y * this.height, this.width, this.height);
+    defineTile(style, x, y) {
+        this.define(style, x * this.width, y * this.height, this.width, this.height);
     }
 
-    draw(name, context, x, y, flip = false) {
-        const buffer = this.tiles.get(name)[flip ? 1 : 0];
+    draw(style, context, x, y, flip = false) {
+        const buffer = this.tiles.get(style)[flip ? 1 : 0];
         context.drawImage(buffer, x, y);
     }
 
-    drawAnim(name, context, x, y, distance) {
-        const animation = this.animations.get(name);
+    drawAnim(style, context, x, y, distance) {
+        const animation = this.animations.get(style);
         this.drawTile(animation(distance), context, x, y);
     }
 
-    drawTile(name, context, x, y) {
-        this.draw(name, context, x * this.width, y * this.height);
+    drawTile(style, context, x, y) {
+        this.draw(style, context, x * this.width, y * this.height);
     }
 }
```

Comment:
In the SpriteSheet it should still be name since style does not mean anything in this context.

Reply:
`drawTile` now uses `name`.